src/makedeb/functions/build_deb.sh

```diff
@@ -1,34 +1,37 @@
 # This function should be run from the directory containing the 'DEBIAN' folder.
+zstdthreads=0
+zstdlevel=10
 build_deb() {
   local pkgname="${1}" \
         tar_control_arguments \
         tar_data_arguments
 
-  # Generate control.tar.gz archive.
+  # Generate control.tar.zst archive.
   cd DEBIAN/
   mapfile -t tar_control_arguments < <(find ./ | grep -v '^\./$')
-  tar -czf ./control.tar.gz "${tar_control_arguments[@]}"
+  tar -cf ./control.tar "${tar_control_arguments[@]}"
+  zstd -T${zstdthreads} -${zstdlevel} --rm -q control.tar
```

Comment:
These variable references should be quoted. It might not cause any issues, but it's still best practice to do so.

Wrap it around the whole string around the first so there isn't a quote in the middle of the argument:

```sh
"-T${zstdthreads}"
```